### PR TITLE
feat(api): add import groups in environment

### PR DESCRIPTION
**Pull request by bnjjj**
Signed-off-by: Benjamin Coenen <benjamin.coenen@hotmail.com>

### CODE REVIEW

File: tests/cli_ctl_environment.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+name: Environement Command TestSuite with CDS ctl
+testcases:
+- name: assert filepath, your current directory must be at the root of this project
+  steps:
+  - script: '[ -f ./tests/fixtures/group_to_add.yml ]'
+  assertions:
+    - result.code ShouldEqual 0
+- name: prepare test
+  steps:
+  - script: {{.cds.build.cds}} project remove --force ITCLIAPPPRJ
```
Comment (yesnault) [inline tests/cli_ctl_environment.yml]: use cdsctl ?

File: cli/cdsctl/environment_group.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+	environmentGroup = cli.NewCommand(environmentGroupCmd, nil,
+		[]*cobra.Command{
+			cli.NewCommand(environmentGroupImportCmd, environmentGroupImportRun, nil),
+		})
+)
+
+var environmentGroupImportCmd = cli.Command{
+	Name:  "import",
+	Short: "Import group linked to a CDS environment",
```
Comment (yesnault) [inline cli/cdsctl/environment_group.go]: Perhaps more details about what is "import" should be more friendly for user?  "delete all and import all."

File: tests/cli_ctl_environment.yml (hunk 1 above)
Comment (bnjjj) [reply tests/cli_ctl_environment.yml]: next pr see this branch https://github.com/ovh/cds/tree/feat_cli_ctl

File: cli/cdsctl/environment_group.go (hunk 2 above)
Comment (bnjjj) [reply cli/cdsctl/environment_group.go]: Not always delete all and import all as explained here https://github.com/ovh/cds/pull/1410/files/2ee7cbf013ebaf915fe99726cb01441e7ca59065#diff-c87bd071ac37a75b0b4d7ac50f2947d6R38

But yes, I agree, we can see together to find better way to explain import to user

Comment (yesnault) [reply cli/cdsctl/environment_group.go]: ok, I have not seen the force flag

**Pull request merged**